File: atcoder/2017/Other/1125_CODE_FESTIVAL_2017_Final/B.py
S = input()
dic = {'a': 0, 'b':0, 'c':0}

for c in S:
    dic[c] += 1

l = [dic['a'], dic['b'], dic['c']]
if max(l)-min(l) <= 1:
    print('YES')
else:
    print('NO')

File: atcoder/2017/Other/1125_CODE_FESTIVAL_2017_Final/B.js
function Main(input) {
  inp = input.split("\n")
  // 整数の入力
  let S = inp[0];
  let dic = {
    'a': 0, 'b': 0, 'c': 0
  };
  for (let i = 0; i < S.length; i++) {
    dic[S.charAt(i)] += 1;
  }
  let l = [dic.a, dic.b, dic.c];
  if (Math.max(...l) - Math.min(...l) > 1) {
    console.log("NO");
  } else {
    console.log("YES");
  }
};

Main(require("fs").readFileSync("/dev/stdin", "utf8"));